daml/Common.daml

```diff
@@ -5,19 +5,19 @@ import Daml.Script
 import DA.Finance.Asset (AssetDeposit)
 import DA.Finance.Types (Id(..), Asset(..), Account(..))
 import DA.Next.Set
-import Marketplace.AssetDescription (AssetDescription(..))
-import qualified Marketplace.Operator as Operator
-import qualified Marketplace.Custodian as Custodian
-import qualified Marketplace.Custody as Custody
-import qualified Marketplace.Issuance as Issuance
-import qualified Marketplace.Trading.Exchange as Exchange
-import qualified Marketplace.Trading.Matching as Matching
+import Marketplace.Claims.AssetDescription (AssetDescription(..))
```

Comment:
I'd say this should go under `Issuance`, not `Claims`. The later has mostly pure functions.

Reply:
I've moved this into `Issuance`